x/contractmanager/client/cli/query_failure.go

```diff
@@ -0,0 +1,74 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/neutron-org/neutron/x/contractmanager/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdAllFailures() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "all-failures",
+		Short: "shows all failures",
+		Args:  cobra.RangeArgs(0, 1),
```

Comment:
Why do you define from 0 to 1 arguments? I don't see any argument handling code further.

Reply:
Forgot to remove it. Thanks for pointing